javascript/code-challenges/queueWithStacks/queue-with-stacks.js

```diff
@@ -0,0 +1,62 @@
+class Stack {
+    constructor() {
+        this.storage = [];
+    }
+}
+
+push(data) {
+    return this.storage.unshift(data)
+}
+
+pop() {
+    return this.storage.shift()
+}
+
+peek() {
+    return this.storage.length ? this.storage[0] : null;
+}
+
+class PseudoQueue {
+    constructor(stack1 = new Stack(), stack2 = new Stack()) {
+        this.stack1 = stack1;
+        this.stack2 = stack2;
+    }
+
+    enqueue(data) {
+        this.stack1.storage.forEach(val => {
+            this.stack2.storage.push(val)
+        })
+        this.stack2.storage.push(data)
+    }
+
+    dequeue() {
+        this.stack1.storage.push(this.stack2.storage.pop())
+        retuen this.stack1.storage[0]
+    }
+}
+
+// Tests
+
```

Comment:
This way of adding tests isn't the way that we would like to see. With next up-coming code challenges could you please add a file in your folder for that code challenges called `__tests__` and then have a file with this living in it. So it would look like this:

```
code-challenges
|
|-----queueWithStacks
                |-----queue-with-stacks.js
                |-----README.md
                |-----__tests___
                              |------queue-with-stacks.test.js